examples/sadtalker/README.md

```diff
@@ -0,0 +1,129 @@
+| --- | --- | --- | --- |
+| BFM_Fitting/ | 3DMM library files. | Infer / Finetune / Train | [BFM_Fitting/](https://download-mindspore.osinfra.cn/toolkits/mindone/sadtalker/BFM_Fitting/) |
+
+
+**3. Face detection and alignment models**
+
+| Model | Description | Required-by | MindSpore Checkpoint |
+| --- | --- | --- | --- |
+| gfpgan_weights/ | Face detection and enhanced models used in `facexlib` and `gfpgan`. | Infer / Finetune / Train | [gfpgan_weights/](https://download-mindspore.osinfra.cn/toolkits/mindone/sadtalker/gfpgan_weights/) |
+
+
+Please download the required weights accordingly (infer/finetune/train). For example, to perform inference, download all weights tagged "finetune". After downloading, the folder should look like this (case-sensitive):
+
+  ```bash
+  checkpoints/
+  ├── BFM_Fitting
+  │   ├── 01_MorphableModel.mat
+  │   ├── BFM09_model_info.mat
+  │   ├── BFM_exp_idx.mat
+  │   ├── BFM_front_idx.mat
+  │   ├── Exp_Pca.bin
+  │   ├── facemodel_info.mat
+  │   ├── select_vertex_id.mat
+  │   ├── similarity_Lm3D_all.mat
+  │   └── std_exp.txt
+  ├── ms
+  │   ├── ms_audio2exp.ckpt
+  │   ├── ms_audio2pose.ckpt
+  │   ├── ms_generator.ckpt
+  │   ├── ms_he_estimator.ckpt
+  │   ├── ms_kp_extractor.ckpt
+  │   ├── ms_mapping.ckpt
+  │   ├── ms_mapping_full.ckpt
+  │   ├── ms_net_recon.ckpt
+  │   ├── ms_vgg19.ckpt # only required for training
+  │   ├── ms_wav2lip.ckpt # only required for training
+  │   └── ms_hopenet_robust_alpha1.ckpt # only required for training
+  gfpgan/
+  └── weights
+      ├── alignment_WFLW_4HG.ckpt
+      ├── detection_Resnet50_Final.ckpt
+      ├── GFPGANv1.4.ckpt
+      └── parsing_parsenet.ckpt
+  ```
+
+ > _NOTE:_  The checkpoint filename or filepath is pre-defined in [sadtalker.yaml](examples\sadtalker\config\sadtalker.yaml). If you want to customize the file name or folder structure, modify the YAML file at the same time.
+
+
+### Training Data
+
+We use [VoxCeleb](https://mm.kaist.ac.kr/datasets/voxceleb/) data to train SadTalker. Training codes is still under developement. We will release it when it's ready, thanks!
+
+
+### Example input data for inference
+
+In the original github, there're some example audios and images under [SadTalker/examples](https://github.com/OpenTalker/SadTalker/tree/main/examples). You can download them to quickly start playing Sadtalker! :wink:
+
+
+## Inference
+
+To generate a talker head video, you have to specify a single portrait image using the argument `--source_image` and an audio file via `--driven_audio`. If you don't specify, it will use the default values.
+
+As reference, you can run the following commands to execute inference process. There're also some arguments to customize the animation, please refer to [input arguments](./utils/arg_parser.py).
+
+```bash
+python inference.py --config ./config/sadtalker.yaml --source_image examples/source_image/people_0.png --driven_audio examples/driven_audio/imagine.wav
+```
+
+## Examples
```

Comment:
Better to organize the sections like. No need to make `Examples` a big section

## Inference
#### Results

Reply:
Thanks, restructured